internal/wasi_snapshot_preview1/logging/logging.go

```diff
@@ -138,7 +138,7 @@ func (i logFilestat) Log(_ context.Context, mod api.Module, w logging.Writer, pa
 		w.WriteString(",size=")              //nolint
 		writeI64(w, le.Uint64(buf[32:]))
 		w.WriteString(",mtim=") //nolint
-		writeI64(w, le.Uint64(buf[40:]))
+		writeU64(w, le.Uint64(buf[40:]))
```

Comment:
prevent logging showing negative time

Reply:
if the negative is found, doesn't that mean mtim can have the 63-th bit set? that sounds weird

Reply:
you mean it is easier to see a problem if we leave it I64?

Reply:
anyway it is better for negative.. as that's for time before 1970. I'm removing it.

Reply:
oh I thought you observed the negative and that's why you changed...

Reply:
heh I did, but then it was while debugging something ;)